docs/CLI.md

```diff
@@ -61,3 +61,12 @@ gulp has very few flags to know about. All other flags are for tasks to use if n
 
 **--log-level**, **-L**
     Set the loglevel. -L for least verbose and -LLLL for most verbose. -LLL is default.
+
+**--config**, **-c**
+    Manually set path of configuration file.
+
+**--ignore-config**
```

Comment:
Do we need this right away?  The config files are an advanced feature that is opt-in by placing the files on the filesystem.  Maybe this shouldn't be added until someone needs it.